mlperf_logging/rcp_checker/training_1.1.0/rcps_bert.json

```diff
@@ -69,51 +69,50 @@
        2777088, 2801664, 2678784, 2801664, 2703360, 2629632, 2727936, 2703360, 2654208, 2949120]
   },
 
-  "bert_ref_4096":
+  "bert_ref_3072":
```

Comment:
Depends on whether #221 or this PR gets merged in first, one of them need to bring this bert rcp change up to `2.0.0`

Reply:
Ack.